openapi.yaml

```diff
@@ -6585,6 +6585,67 @@ paths:
       - lang: CLI
         source: >
           linode-cli lke pool-delete 12345 456
+  /lke/clusters/{clusterId}/api-endpoints:
+    parameters:
+    - name: clusterId
+      in: path
+      description: ID of the Kubernetes cluster to look up.
+      required: true
+      schema:
+        type: integer
+    x-linode-cli-command: lke
+    get:
+      operationId: getLKEClusterAPIEndpoints
+      x-linode-cli-action: api-endpoints-list
+      security:
+      - personalAccessToken: []
+      - oauth:
+        - lke:read_only
+      tags:
+      - Linode Kubernetes Engine (LKE)
+      servers:
+      - url: https://api.linode.com/v4beta
+      summary: List Kubernetes API Endpoints
+      description: >
+        List the Kubernetes API server endpoints for this cluster. Please note that it often takes
+        2-5 minutes before the endpoint is ready after first [creating a new cluster](/api/v4/lke-clusters/#post).
+
+
+        **Beta**: This endpoint is in private beta. Please make sure to prepend all requests with
+        `/v4beta` instead of `/v4`, and be aware that this endpoint may receive breaking
+        updates in the future. This notice will be removed when this endpoint is out of
+        beta. Sign up for the beta [here](https://welcome.linode.com/lkebeta/).
+      responses:
+        '200':
+          description: Returns the Kubernetes API server endpoints for this cluster.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
```

Comment:
In addition to renaming this endpoint, the return structure changed to be more in line with the rest of the API.
Now the response looks something like this
```
{
  "results": 1,
  "pages": 1,
  "page": 1,
  "data": [{"endpoint": "https://207.xxx.xxx.xxx:443"},{"endpoint": "https://207.xxx.xxx.xxx:6443"}],
}
```